examples/notebooks/Differential_Dependencies.ipynb

```diff
@@ -0,0 +1,3002 @@
+              "      <td id=\"T_0791a_row9_col4\" class=\"data row9 col4\" >622</td>\n",
+              "      <td id=\"T_0791a_row9_col5\" class=\"data row9 col5\" >61</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th id=\"T_0791a_level0_row10\" class=\"row_heading level0 row10\" >10</th>\n",
+              "      <td id=\"T_0791a_row10_col0\" class=\"data row10 col0\" >DP 261</td>\n",
+              "      <td id=\"T_0791a_row10_col1\" class=\"data row10 col1\" >2024-03-06</td>\n",
+              "      <td id=\"T_0791a_row10_col2\" class=\"data row10 col2\" >Moscow (VKO)</td>\n",
+              "      <td id=\"T_0791a_row10_col3\" class=\"data row10 col3\" >Kaliningrad (KGD)</td>\n",
+              "      <td id=\"T_0791a_row10_col4\" class=\"data row10 col4\" >1059</td>\n",
+              "      <td id=\"T_0791a_row10_col5\" class=\"data row10 col5\" >144</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th id=\"T_0791a_level0_row11\" class=\"row_heading level0 row11\" >11</th>\n",
+              "      <td id=\"T_0791a_row11_col0\" class=\"data row11 col0\" >DP 536</td>\n",
+              "      <td id=\"T_0791a_row11_col1\" class=\"data row11 col1\" >2024-03-05</td>\n",
+              "      <td id=\"T_0791a_row11_col2\" class=\"data row11 col2\" >Kaliningrad (KGD)</td>\n",
+              "      <td id=\"T_0791a_row11_col3\" class=\"data row11 col3\" >Saint Petersburg (LED)</td>\n",
+              "      <td id=\"T_0791a_row11_col4\" class=\"data row11 col4\" >798</td>\n",
+              "      <td id=\"T_0791a_row11_col5\" class=\"data row11 col5\" >92</td>\n",
+              "    </tr>\n",
+              "  </tbody>\n",
+              "</table>\n"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 8
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "We can notice that the distance is between 0 and 50. Therefore, the DD\n",
+        "\"Departure [0, 0] ; Arrival [0, 0] -> Distance [0, 50]\" holds in the table."
+      ],
+      "metadata": {
+        "id": "ypr5O1tc2Kuo"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Second DD explanation"
+      ],
+      "metadata": {
+        "id": "OM5rQoe02L5R"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Now let's move to the second DD: \"Distance [0, 50] -> Duration [0, 15]\". This DD means the following: for any\n",
+        "pair of tuples if the distance between them on the column \"Distance\" is between 0 and 50, then the distance on\n",
+        "the column \"Duration\" is between 0 and 15. In other words, if two flights have similar distances, then they\n",
+        "last for a similar time. As can be seen from the table, almost all flights have similar distances which differ\n",
```

Comment:
discussed this issue in voice chat (we are showing all suitable records for the first record); fix in example too?